lib/lib_rss.php

```diff
@@ -170,7 +170,7 @@ function format_bytes($bytes, $precision = 2, $system = 'IEC') {
 	$pow = $bytes === 0 ? 0 : floor(log($bytes) / log($base));
 	$pow = min($pow, count($units) - 1);
 	$bytes /= pow($base, $pow);
-	return format_number($bytes, $precision) . ' ' . $units[$pow];
+	return format_number($bytes, $precision) . ' ' . $units[$pow];
```

Comment:
I've changed this space because it wasn't a regular space. I don't know if it was on purpose or not. The space was coded on two bytes and broke the cli display layout.

Reply:
It was on purpose, but if it broke anything, it is not a problem to remove it

Reply:
What was the use case which needed such space?

Reply:
thin non-breaking space, if I remember correctly, which is the typographic standard between a number and its unit.